scripts/_1_1_3_4d_probe_responses.py

```diff
@@ -0,0 +1,264 @@
+
+        for (
+            scenario,
+            target_field,
+            kdma_type,
+            aligned_kdma_field,
+            baseline_kdma_field,
+            aligned_probes_field,
+            baseline_probes_field,
+        ) in scenarios:
+            aligned_probes_raw = doc.get(aligned_probes_field, [])
+            aligned_probes_parsed = parse_probe_responses(aligned_probes_raw, kdma_type)
+
+            baseline_probes_raw = doc.get(baseline_probes_field, [])
+            baseline_probes_parsed = parse_probe_responses(
+                baseline_probes_raw, kdma_type
+            )
+
+            aligned_doc = {
+                "evalNumber": 11,
+                "evalName": "Phase 2 4D Experiment",
+                "pid": pid,
+                "scenario": scenario,
+                "alignment_target": str(doc.get(target_field)), 
+                "adm_name": doc.get("admName"),  
+                "synthetic": True, 
+                "probe_ids": [
+                    list(pr.keys())[0] for pr in aligned_probes_parsed
+                ],  
+                "probe_responses": aligned_probes_parsed,
+                "results": {
+                    "ta1_session_id": pid,
+                    "kdmas": [
+                        {
+                            "kdma": (
+                                scenario[9:11].lower()
+                                if scenario[9:11] == "AF"
+                                else (
+                                    "merit"
+                                    if scenario[9:11] == "MF"
+                                    else (
+                                        "personal_safety"
+                                        if scenario[9:11] == "PS"
+                                        else "search"
+                                    )
+                                )
+                            ),
+                            "value": doc.get(aligned_kdma_field),
+                        }
+                    ],
+                },
+            }
+            adm_target_runs.insert_one(aligned_doc)
+            total_docs_created += 1
+
+            # Create baseline document
+            baseline_doc = {
+                "evalNumber": 11,
+                "evalName": "Phase 2 4D Experiment",
+                "pid": pid,
+                "scenario": scenario,
+                "alignment_target": str(doc.get(target_field)),  # Convert to string!
+                "adm_name": "ALIGN-ADM-OutlinesBaseline-Mistral-7B-Instruct-v0.3",  # Add for consistency
+                "synthetic": True,  # Add this field
+                "probe_ids": [
+                    list(pr.keys())[0] for pr in baseline_probes_parsed
+                ],  # Add probe_ids!
+                "probe_responses": baseline_probes_parsed,
+                # Add results structure like eval 14
+                "results": {
+                    "ta1_session_id": pid,
```

Comment:
Same.  If we don't have (and don't need) the TA1 session ID, maybe we leave it out, or put "N/A" (for not available), but I'm not seeing why we'd put the `pid` here.